homeassistant/components/media_player/vizio.py

```diff
@@ -32,7 +32,7 @@
 )
 from homeassistant.helpers import config_validation as cv
 
-REQUIREMENTS = ['pyvizio==0.0.2']
+REQUIREMENTS = ['pyvizio==0.0.2', 'urllib3==1.22']
```

Comment:
You should not have to add a dependency on urllib3, it should get installed as a dependency of requests 2.16 and later (when they unvendored all the things).